patches/spec.fetched.json/10-fix-virtual-circuit-attr-in-interconnection-port.patch

```diff
@@ -1,84 +0,0 @@
-diff --git a/spec/oas3.patched/openapi/public/components/schemas/InterconnectionPort.yaml b/spec/oas3.patched/openapi/public/components/schemas/InterconnectionPort.yaml
```

Comment:
This patch should have been removed in an #25, but wasn't.

Two things to note around this to build confidence in the change:
1. Removing the patch in this PR had no impact on the code
2. The code did not materially change as a result of the sync job in question (the declaration of the virtual_circuits attribute moved to a different line, but the code remained the same): https://github.com/equinix-labs/metal-java/pull/106/files#diff-a7beb2dfd7e5119ab3995f04427d59ffd0e9304ce16294661fae1a027c5d6e9a